scripts/wl-build.sh

```diff
@@ -36,6 +42,12 @@ export NM=${WASI_SDK_ROOT}/bin/llvm-nm
 export AR=${WASI_SDK_ROOT}/bin/llvm-ar
 export RANLIB=${WASI_SDK_ROOT}/bin/llvm-ranlib
 
+if ! builtin type -P wasm-opt
+then
+    logStatus "Using wasm-opt wrapper from ${WASMLABS_REPO_ROOT}/scripts/wrappers"
+    export PATH="$PATH:${WASMLABS_REPO_ROOT}/scripts/wrappers"
+fi
```

Comment:
I think this might have different behavior locally than in a container.

Within a build container, we have `ENV PATH="/opt/priority-bin:$PATH:/opt/bin"` on the `Dockerfile.wasm-base`, what will always find our wrapper `wasm-opt` script.

If we don't put our wrapper to be found earlier than the original `wasm-opt`, clang will call to wasm-opt with their own optimization flags, what might break a later wasm-opt Asyncify pass done manually on a second wasm-opt run.

Reply:
That's why I do this only if `wasm-opt` is not found on PATH at all. But you're right that it sounds a bit fishy. I will reverse it.